src/app/docs/unity/defaults/defaults.component.html

```diff
@@ -15,35 +15,76 @@ <h2 id="registry">Registry</h2>
 			The Registry is an easy way to use core functionality from the Runtime. It
 			simplifies how they're called and needs less information from the user.
 		</p>
+		<p>
+			NOTE: If Async is enabled in the Ecsact settings the default Registry will
+			be NULL
+		</p>
 	</section>
 	<section>
 		<h2 id="runner">Runner</h2>
 		<p>
-			The Runner stores and executes actions and runs the Ecsact execution. You
-			can choose the following options:
+			The Runner stores and executes its options that are explained more below.
+			There are two types of runners. Both has synonymous functionality with a
+			few key differences:
+		</p>
+		<h3>Default Runner</h3>
+		<p>
+			Runs the Ecsact execution synchronously. You can choose the following
+			options:
 		</p>
 		<ol>
 			<li><code>Default</code>: Executes per Update tick</li>
 			<li><code>Fixed Update</code>: Executes per FixedUpate tick</li>
 			<li><code>None</code>: Implement your own Runner</li>
 		</ol>
+		<h3>Async Runner</h3>
+		<p>
+			Runs the Ecsact execution asynchronously. If you're interested in
+			developing multiplayer games this is the way to go. There are some
+			differences in how it's used.
+		</p>
+		<ul>
+			<li>
+				the Default Registry is NULL which comes at the cost of direct access to
+				its inner workings
+			</li>
+			<li>
+				Instead of choosing how the Ecsact execution functions, you instead
+				connect to the reference with a <code>delta speed</code>. This is the
+				time in milliseconds it takes for systems to be executed
```

Comment:
> connect to the reference

This is very ambiguous. I know you're talking about the refernece implementation, but the user will not understand.

Instead I would write something like this:

> you instead _connect_ to an async execution. The implementation of the async runtime (maybe link to the async module documentation) determines how fast your systems are executed. By default the Ecsact unity integration uses the **reference async runtime** which allows you to specify a fixed deltatime in the connection query parameter as milliseconds. (Maybe insert example here)